core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/NotificationsApiService.scala

```diff
@@ -1,15 +1,30 @@
 package org.broadinstitute.dsde.rawls.webservice
 
-import org.broadinstitute.dsde.rawls.model.{WorkspaceName, Notifications}
-import org.broadinstitute.dsde.rawls.model.Notifications.{WorkspaceNotificationType, WorkspaceNotification, NotificationType}
+import org.broadinstitute.dsde.rawls.model.{Notifications, UserInfo, WorkspaceName}
+import org.broadinstitute.dsde.rawls.model.Notifications.{NotificationType, WorkspaceNotification, WorkspaceNotificationType}
+import org.broadinstitute.dsde.rawls.workspace.WorkspaceService
+import org.broadinstitute.dsde.rawls.openam.UserInfoDirectives
 import spray.routing.HttpService
 import spray.httpx.SprayJsonSupport._
 import spray.json.DefaultJsonProtocol._
 
+import scala.concurrent.ExecutionContext
 /**
  * Created by dvoet on 3/28/17.
  */
-trait NotificationsApiService extends HttpService {
+trait NotificationsApiService extends HttpService with PerRequestCreator with UserInfoDirectives {
+  implicit val executionContext: ExecutionContext
+
+  val workspaceServiceConstructor: UserInfo => WorkspaceService
+  //this route requires UserInfo
+  val sendChangeNotificationRoute = requireUserInfo() { userInfo =>
+    path("notifications"/ "workspace" / Segment / Segment / "sendChangeNotification") { (namespace, name) =>
```

Comment:
Actually, does this belong in the notifications service or the workspace service? I think the notifications service is for managing notifications overall whereas this is for sending a specific notification related to a specific workspace. I'm inclined to say this belongs in the workspace service, which would match what was outlined in the design document.